modules/govuk_mysql/manifests/xtrabackup/packages.pp

```diff
@@ -0,0 +1,21 @@
+# == Class: Govuk_mysql::Xtrabackup::Packages
+#
+class govuk_mysql::xtrabackup::packages (
+  $s3gof3r_latest  = 'https://github.com/rlmcpherson/s3gof3r/releases/download/v0.5.0/gof3r_0.5.0_linux_amd64.tar.gz',
```

Comment:
I think we should package this as otherwise the owner of that repo could replace that version without our knowledge.

Perhaps it's a good opportunity to try something like FPM for packaging instead of our debhelper scripts?